src/main/java/uk/gov/hmcts/reform/sscs/model/service/hearingvalues/ServiceHearingValues.java

```diff
@@ -93,4 +93,6 @@ public class ServiceHearingValues {
     private List<Vocabulary> vocabulary;
 
     private String hmctsServiceID;
+
+    private boolean caseInterpreterRequiredFlag;
```

Comment:
Could you please update the tests so it that checks the value of caseInterpreterRequiredFlag is coming through correctly?

Reply:
Done